core/src/commands/on_runtime_upgrade.rs

```diff
@@ -61,6 +61,10 @@ pub struct Command {
     #[clap(long, default_value = "false", default_missing_value = "true")]
     pub no_weight_warnings: bool,
 
+    /// Whether to enforce the new runtime `spec_version` is greater or equal to the existing
+    /// `spec_version`.
+    #[clap(long, default_value = "true", default_missing_value = "true")]
+    pub check_spec_version: bool,
```

Comment:
```suggestion
    #[clap(long)]
    pub disable_check_spec_version: bool,
```

These are flags and you should not pass any `true` or `false`. So make it more clear by naming it `disable-check-spec-version`.